ss-genuri.py

```diff
@@ -20,5 +22,12 @@
     config['server_port'],
 )
 
-encoded_uri = base64.b64encode(uri.encode('utf-8'))
-print('ss://%s' % encoded_uri.decode('utf-8'))
+encoded_uri = base64.b64encode(uri.encode('utf-8')).decode('utf-8')
+
+remarks = 'Shadowsocks Server'
```

Comment:
How about we change lines 27-32 to:
```
remarks = config.get('remarks', 'Shadowsocks Server')
remarks = "".join(remarks.split(' '))
print('ss://%s%s' % (encoded_uri, remarks))
```
This will give us python2 support, heh.

Reply:
Thank you for the input. I will test both versions with this update.